tests/integration/520.command.link.test.mjs

```diff
@@ -0,0 +1,47 @@
+import { join } from 'path'
+import process from 'process'
+
+import { expect, test } from 'vitest'
+
+import { isFileAsync } from '../../src/lib/fs.mjs'
+
+import callCli from './utils/call-cli.cjs'
+import { getCLIOptions, withMockApi } from './utils/mock-api.cjs'
+import { withSiteBuilder } from './utils/site-builder.cjs'
+
+test('should create gitignore in repository root when is root', async () => {
+  await withSiteBuilder('repo', async (builder) => {
+    await builder.withGit().buildAsync()
+
+    await withMockApi(
+      [],
+      async ({ apiUrl }) => {
+        await callCli(['link'], getCLIOptions({ builder, apiUrl }))
+
+        expect(await isFileAsync(join(builder.directory, '.gitignore'))).toBe(true)
+      },
+      true,
+    )
+  })
+})
+
+test.skipIf(process.platform === 'win32')(
```

Comment:
This was skipped before too, but from what I looked at seems like a problem wither in how we use netlify/config here or a problem inside netlify/config on windows.